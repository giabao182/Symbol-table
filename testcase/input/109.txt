INSERT a number
INSERT b string
ASSIGN a 42
ASSIGN b 'hello'
ASSIGN b 42
ASSIGN a 'wrong'