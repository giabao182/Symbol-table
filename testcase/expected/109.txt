success
success
success
success
TypeMismatch: ASSIGN b 42
TypeMismatch: ASSIGN a 'wrong'